API.py

```diff
@@ -0,0 +1,49 @@
+import requests
+
+
+def requests_top_repos(language: str, num_of_repos: int):
+    """
+    The function requests and print the popular repositories in GitHub by language
+    :param language:The programing language of the repos
+    :param num_of_repos: Count of repos to request
+    """
+    url = f"https://api.github.com/search/repositories?q=language:{language}&per_page:{num_of_repos}&sort:stars"
+    ans = requests.get(url)
+    for item in ans.json()['items']:
+        print(f"{item['name']} is a {language} repo with {item['watchers']} stars.")
```

Comment:
The request and the print should not be on the same function - you care coupling IO with logic 
https://wiki.c2.com/?SeparateIoFromCalculation